packages/metrics-app/src/services/measurements.ts

```diff
@@ -74,7 +91,19 @@ export async function performMeasurementsFromQuery(query: FCMDataMessage): Promi
     // Check location
     const coordinates = await getCurrentCoordinates()
     if (!coordinates) {
-      logger.warn(TAG, 'Aborted performing measurements as coordinates cannot be obtained')
+      logger.warn(
+        TAG,
+        'Aborted performing measurements as coordinates cannot be obtained'
+      )
+      return
+    }
+    // Check area of location
+    const area = await getAddress()
+    if (!area) {
```

Comment:
The area is not strictly necessary, so let's not abort the measurement if it is missing.